src/Pages/Competition.jsx

```diff
@@ -60,11 +61,14 @@ function Competition({ dogs, competitions }) {
       <header>
         <h1>{competitionTitle.exhibitionName}</h1>
       </header>
-      <aside>
-        <button onClick={handleClick}>Buy Tickets</button>
-        <button onClick={handleClick2}>Register for Competition</button>
+      <aside className="two-column-grid ">
+        <>
+        <button className="comp-btn" onClick={handleClick}>Buy Tickets</button>
+        </>
+        <button className="comp-btn" onClick={handleClick2}>Register for Competition</button>        
```

Comment:
You don't need to change this, but generally numbering similar functions is frowned on by devs. Something like `handleBuyClick` and `handleRegisterClick` would've been a better choice here